app/src/main/java/org/wikipedia/readinglist/ReadingListBehaviorsUtil.kt

```diff
@@ -205,6 +225,41 @@ object ReadingListBehaviorsUtil {
             .show()
     }
 
+    fun showDeleteListsUndoSnackbar(activity: Activity, readingLists: List<ReadingList>?, callback: SnackbarCallback) {
+        if (readingLists == null) {
+            return
+        }
+        val snackBar = FeedbackUtil.makeSnackbar(activity, getDeleteListMessage(activity, readingLists))
+        if (!(readingLists.size == 1 && readingLists[0].isDefault)) {
```

Comment:
Would it be possible to filter the default list so that the remaining logic can be skipped?

Reply:
The logic here is, do not show an undo button if -> default is the only button selected for deletion. However, if it is just one of many, then undo is shown, but upon clicking undo, default should not get created again. So a general filter might not work. Lmk what you feel about it.